static/header.less

```diff
@@ -145,7 +145,7 @@
                 &:before {
                     content: '';
                     position: absolute;
-                    bottom: 20px;
+                    bottom: 8px;
```

Comment:
Can the other `bottom: 8px;` be removed now?

Reply:
yes, pushed